capstone/client/src/components/ListPage.js

```diff
@@ -49,6 +49,8 @@ class ListPage extends Component {
       listId: -1,
       listName: null,
       userId: -1,
+      displayZipCodeInput: false,
```

Comment:
do you need this field? seems like location = null is enough?

Reply:
I had it like that first. Doing it that way made it so the zip code box would appear on page load since location starts as null and it would disappear after a user shares their location. I added this field because I wanted so I could have this zip code box be hidden on load and appear if the user doesn't share their location.